tests/cpu/transforms/BuiltinTransform_tests.cpp

```diff
@@ -392,6 +392,8 @@ AllValues UnitTestValues
         { { 0.125f, 0.1f, 0.075f }, { 0.211320835792f, 0.149169650771f, 0.085452970479f } } },
     { "ACES-LMT - BLUE_LIGHT_ARTIFACT_FIX",
         { { 0.5f, 0.4f, 0.3f }, { 0.48625676579f,  0.38454173877f,  0.30002108779f } } },
+    { "ACES-LMT - GAMUT_MAP_13",
+        { { 0.5f, 0.4f, 0.3f }, { 0.50000000000f,  0.40000000596f,  0.30000001192f } } },
```

Comment:
The {0.5,0.4,0.3} is not a good test value.  The goal of these tests is just a quick test that it's working, but an identity op would pass this test.  Suggest using a value such as { 0.5, 0.4, -0.3 }, or some other color that would actually get compressed.

Reply:
Yes agreed, updated.